scaling_performance/cluster_limits.adoc

```diff
@@ -100,3 +100,102 @@ If you increase the number of nodes to 20, then the pod distribution changes to
 ----
 2200 / 20 = 110
 ----
+
+[[scaling-performance-planning-your-environment-according-to-application-requirements]]
+== Planning Your Environment According to Application Requirements
+
+You might have a deployment such as {product-title} running on virtual machines
+(VMs) on top of Red Hat OpenStack Platform with application requirements such
+as:
+
+[options="header",cols="6"]
+|===
+|Pod Type |Pod Quantity |Max Memory |CPU Cores |Persistent Storage |Network Bandwidth
+
+|apache
+|100
+|500MB
+|0.5
+|1GB
+|100qps
+
+|node.js
+|200
+|1GB
+|1
+|1GB
+|100qps
+
+|postgresql
+|100
+|1GB
+|2
+|10GB
+|20Mbit
+
+|JBoss EAP
+|100
+|1GB
+|1
+|1GB
+|5Mbit
+|===
+
+Extrapolated requirements: 550 CPU cores, 450GB RAM, 1.4TB storage and < 1Gbit
+network.
+
+To fulfill these resource requirements, a highly available topology might
```

Comment:
Drop this first table that discusses the control plane topology.  This is documented elsewhere.  No need to duplicate.